#! /bin/sh /usr/share/dpatch/dpatch-run
## baseline-symbols-update.dpatch by Otavio Salvador <otavio@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Update the library symbols to detect ABI changes.

@DPATCH@
diff -urNad parted-1.7.1~/scripts/data/abi/baseline_symbols.txt parted-1.7.1/scripts/data/abi/baseline_symbols.txt
--- parted-1.7.1~/scripts/data/abi/baseline_symbols.txt	2006-05-29 09:56:17.000000000 -0300
+++ parted-1.7.1/scripts/data/abi/baseline_symbols.txt	2006-05-29 09:56:25.000000000 -0300
@@ -28,6 +28,7 @@
 FUNC:ext2_commit_metadata
 FUNC:ext2_copy_block
 FUNC:ext2_destroy_dev_handle
+FUNC:ext2_determine_itoffset
 FUNC:ext2_do_inode
 FUNC:ext2_find_free_block
 FUNC:ext2_find_free_inode
@@ -291,6 +292,8 @@
 FUNC:ped_disk_print
 FUNC:ped_disk_probe
 FUNC:ped_disk_remove_partition
+FUNC:ped_disk_s390_done
+FUNC:ped_disk_s390_init
 FUNC:ped_disk_set_partition_geom
 FUNC:ped_disk_sun_done
 FUNC:ped_disk_sun_init
@@ -408,6 +411,9 @@
 FUNC:ped_unit_set_default
 FUNC:ped_unregister_disk_type
 FUNC:raw_part_parse
+FUNC:s390_alloc
+FUNC:s390_clobber
+FUNC:s390_probe
 NOTYPE:__bss_start
 NOTYPE:_edata
 NOTYPE:_end
